engine/src/cmd/ai/aggressive.cpp

```diff
@@ -173,7 +148,7 @@ static AIEvents::ElemAttrMap *getLogicOrInterrupt(string name,
         vsUMap<string, AIEvents::ElemAttrMap *> &mymap,
         int personalityseed) {
     string append = "agg";
-    static vsUMap<string, string> myappend = getAITypes();
```

Comment:
I thought we discussed previously that the AI personality types are still needed. Is there code elsewhere that supports them? If so, I don't see it.

Reply:
This line change should not be in this PR. I'll revert.

Reply:
Reverted.